packages/server/src/resources/resources.controller.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  CACHE_MANAGER,
+  Controller,
+  Get,
+  HttpService,
+  Inject,
+  Param,
+} from '@nestjs/common';
+import { Cache } from 'cache-manager';
+import { CourseModel } from '../course/course.entity';
+import { Connection } from 'typeorm';
+
+/**
+ * Controller for any public resources on the app. Anything accessed through this controller does
+ * not need any JWT or role guards.
+ */
+@Controller('resources')
+export class ResourcesController {
+  constructor(
+    private connection: Connection,
+    private httpService: HttpService,
+    @Inject(CACHE_MANAGER) private cacheManager: Cache,
+  ) {}
+
+  @Get('/calendar/:course/refresh=:refresh')
```

Comment:
also can we then deprecate [this endpoint](https://github.com/sandboxnu/office-hours/blob/d14a6d9848c177d784ab82cadcc73a942ee3e08d/packages/server/src/course/course.controller.ts#L465) and update the cron job that re-scrapes the calendar endpoints? (or make a new cron job to parse and cache these at midnight